packages/nextjs/tsup.config.ts

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'tsup';
+
+import { name, version } from './package.json';
+
+export default defineConfig(overrideOptions => {
+  const isProd = overrideOptions.env?.NODE_ENV === 'production';
+
+  return {
+    entry: ['src/index.ts', 'src/ssr/index.ts', 'src/server/index.ts', 'src/api/index.ts', 'src/client/index.ts'],
+    minify: isProd,
+    sourcemap: true,
+    format: ['cjs', 'esm'],
+    // Uncomment this to get metafiles for esbuild bundle analysis
+    // metafile: true,
```

Comment:
Can we add a npm script to build and analyze?
EG: `npm run build:analyze` or similar (up to you) 

We can make this standard across our packages.